src/lib/dapp/DAppSigner.ts

```diff
@@ -106,4 +159,47 @@ export class DAppSigner implements Signer {
   ): Promise<OutputT> {
     throw new Error('Method not implemented.')
```

Comment:
call needs to be implemented to support `transaction.executeWithSigner(signer)` https://github.com/hashgraph/hedera-sdk-js/blob/65a2323f8d9281d686c2f1c4d337b7a22a3ca97c/src/Executable.js#L500